examples/default/copies.json

```diff
@@ -0,0 +1,47 @@
+{
+  "@context": "https://github.com/Inria-Visages/blob/master/BIDS-prov/context.json",
+  "@id": "http://example.org/ds00000X",
+  "generatedAt": "2020-03-10T10:00:00",
+  "wasGeneratedBy": {
+      "@id": "INRIA",
+      "@type": "Project",
+      "startedAt": "2016-09-01T10:00:00",
+      "wasAssociatedWith": { "@id": "NIH",
+                              "@type": "Organization",
+                              "hadRole": "Funding"
+                            }
+    },
+  "@graph": [
+    {
+      "prov:Activity" : [
+        {
+          "@id" : "copy",
+          "wasAssociatedWith" : {
+            "@type" : "prov:SoftwareAgent",
+            "RRID" : "https://scicrunch.org/resources/Any/record/nlx_144509-1/SCR_007037/resolver?q=SPM&l=SPM",
+            "label" : "SPM" 
+          },
+          "startedAtTime" : "2019-10-10T10:00:00",
+          "endedAtTime" : "2019-10-10T10:00:00",
+          "used" : "$HOME/BIDS-prov/nidm-results-examples/spm_default/ds011/sub-01/func/sub-01_task-tonecounting_bold.nii.gz"
+        }
+      ],
+      "prov:Entity" : [
+        {
+          "@id": "$HOME/BIDS-prov/nidm-results-examples/spm_default/ds011/sub-01/func/sub-01_task-tonecounting_bold.nii.gz",
```

Comment:
The path to the file should be a `prov:atLocation` attribute and we can again use a UUID as identifier, something like:
```
"@id": "sjhgdfhjsgd63q54673gbdsjh23yq6564",
"prov:atLocation": "$HOME/BIDS-prov/nidm-results-examples/spm_default/ds011/sub-01/func/sub-01_task-tonecounting_bold.nii.gz",
```